### ISSUE TITLE
Extracting Validation loss.

### ISSUE THREAD

**Issue by Worulz** [jwyang/faster-rcnn.pytorch#245]
@jwyang is it possible to extract validation loss from this? 

https://github.com/jwyang/faster-rcnn.pytorch/blob/baa0385c0ec8cbd56fa204b1eea6aa15fe9ff0ea/lib/model/faster_rcnn/faster_rcnn.py#L49-L114

I'm having trouble extracting the loss from here

https://github.com/jwyang/faster-rcnn.pytorch/blob/baa0385c0ec8cbd56fa204b1eea6aa15fe9ff0ea/lib/model/faster_rcnn/faster_rcnn.py#L53-L60

**Comment by Worulz** [comment]
I've been able to extract the validation loss by changing the shape of input of from roibatch layer then changing options where self.training is shown. Is it okay to feed in ground through labels?. It won't train if I've set the data to no grad and network to eval correct?

**Comment by Worulz** [comment]
Closed. Stale

**Comment by amajee11us** [comment]
Hi @Worulz , can you please point me to the changes that you implemented to get the validation losses during training. I would appreciate your help.
